widgets/post-carousel/tpl/carousel-post-loop.php

```diff
@@ -6,18 +6,26 @@
 while($posts->have_posts()) : $posts->the_post(); ?>
 	<div class="sow-carousel-item" tabindex="-1">
 		<div class="sow-carousel-thumbnail">
-			<?php if( has_post_thumbnail() ) : $img = wp_get_attachment_image_src(get_post_thumbnail_id(), $instance['image_size']); ?>
-				<a href="<?php the_permalink() ?>" style="background-image: url(<?php echo sow_esc_url($img[0]) ?>)" aria-labelledby="sow-carousel-id-<?php echo the_ID(); ?>" tabindex="-1">
+			<?php if ( has_post_thumbnail() ) : $img = wp_get_attachment_image_src( get_post_thumbnail_id(), $instance['image_size'] ); ?>
+				<a href="<?php the_permalink() ?>"
+					<?php echo $link_target == 'new' ? 'target="_blank" rel="noopener noreferrer"' : '' ?>
```

Comment:
Do we want `noopener noreferrer` to be the default here? I can understand that we use this for links inside the WordPress admin, but is it our place to block a referrer in a user's frontend? They might want/need this information to be passed.

Reply:
Replied over Slack